src/magellan_core/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ add_message_files(
   ObstacleStamped.msg
   ObstacleStampedArray.msg
   EncoderCount.msg
+  Int32XYArr.msg
```

Comment:
Per discussion with @camkisailus let's please just have an array of [Points](http://docs.ros.org/melodic/api/geometry_msgs/html/msg/Point.html) for the line points.

Reply:
@Sridhar701Pitt Is claiming this

Reply:
@camkisailus @Sridhar701Pitt please fix this